tide_media.install

```diff
@@ -442,3 +443,59 @@ function tide_media_update_8013() {
     }
   }
 }
+
+/**
+ * Enable Focal Point with default config.
+ */
+function tide_media_update_8014() {
+  if (!\Drupal::moduleHandler()->moduleExists('focal_point')) {
+    /** @var \Drupal\Core\Extension\ModuleInstallerInterface $module_installer */
+    $module_installer = \Drupal::service('module_installer');
+    $module_installer->install(['crop']);
```

Comment:
Isn't `crop` already a dependency of `focal_point`?

Reply:
It is and also added in the info file as well - [Info file](https://git.drupalcode.org/project/focal_point/-/blob/8.x-1.x/focal_point.info.yml). But couldn't find in composer.json and had some issues during installation. So added it to our project.